packages/web/src/pages/nfts/[chainId]/[contractAddress]/[tokenId].tsx

```diff
@@ -43,17 +43,28 @@ const NFTPage: NextPage<NFTPageProps> = ({ nft }) => {
 
 export default NFTPage;
 
-export const getServerSideProps: GetServerSideProps = async (context) => {
+export const getStaticProps: GetStaticProps = async (context) => {
```

Comment:
This is big change

This is called only
- 300 seconds passed after the last request

Reply:
I think I can add...

- if the record does not exists, revalidate faster

Reply:
so only nft data (metadata) are synced 300 seconds each,  and order data are refreshed realtime, right?
looks good to me